sim/midas/src/main/cc/bridges/synthesized_prints.h

```diff
@@ -83,7 +83,7 @@ class synthesized_prints_t : public bridge_driver_t {
   const int printno;
 
   // DMA batching parameters
-  const size_t beat_bytes = DMA_DATA_BITS / 8;
+  const size_t beat_bytes = BridgeConstants::STREAM_WIDTH_BYTES;
```

Comment:
Can this be `constexpr` now also? 
```suggestion
  constexpr size_t beat_bytes = BridgeConstants::STREAM_WIDTH_BYTES;
```